### PR TITLE
Colorbar bugfixing

### PR DESCRIPTION
**Pull request by dsp0011**

### CODE REVIEW

File: src/demo/example-data/new-layered-map.json [hunk 1]
```diff
@@ -23,18 +23,44 @@
...
...57naa8/dF5h4ydZ+lDg8Mwa2GnUVAWoypysOpy6F4iudHLN6XAe2nyMhARqNNT/7cQesrDVfImw0Uzp9R0GLCo++alJ5CwlQKAQmFIRxENfY2rXCbX8QQWW4xZdZTiMRjQS1sMKfnUgWKkHmNBtRIQW6xGTZE64oggs2yFxPa/RbAhoPYZIfj7J0iG89R31UHgvXGkxZRvupXH9jYjQMCco1MsOxqrzhc0v53nKYhIq4tNUWzZOnf1clCVcc7P/13tr1qVEMoUELORcoOvH9BOZRTLwA1jIurrb72Vulfj1EVP7tbw8QPsdV1YLTXxG7IBhH/zSYwcXbXVDD1Np5Xut6G5YcMh7Bk0BzaPNsjmvYYMmOCJ5CmRosYtIKFfLILDsRs1VU2iqgBq9pJ0IJQdtDvWv/99eFNSGd8KVl7ImDxNR12iPDUGFfwVWYKaEjUiq841lL7bN1qbQQTO7hc5zp/DZyuo+f4cWYVgZ5aAvNo7YmSG10WD28fkevbLY7KMZXlg9uq5269qCu37PSgPoLHpOvzMmoDe0k4DGGxG9rY792/EMWeAfhYA7cR9RU0mRcDLAhZ0htJFFqIYVVtT/2xxmbiYQmD0s8aItkS2kUMH7+LydSURiEz8z6DI1gvivBpb3s4Oyc9sj89EKlRTQbKWQgv22Vaudk4hts51OW1IMMA+ig9fhAGRJcpF8QWt8UdU1adfb+zAuvqzB0NaLrPEjNusTFcmEJTpCPMUCD9pju7apTNWPCi9qLU7tavuwWVoKVkKW+7vR9hHk+GnsqFeYO49IQUhST0RiWnNM0RW5AMTX8xOW8NLQhQz7yLSNCe1f3d3D1fqk8M7rQczuPPqBwLgaysg4A11Ljhd3vsPrbM9T0agAK9fQxIaJqCPLFaoEpihhSqwGiQAwiEiQUwrOS8o2fkwLwprPdqHCyjl27hv/zc8lhmcX6xHr7QObxsvzhRlwYOgYs4nZ7536DVqva4wgK53/J00XyjkilwsqJdrPZ4145cqWOWk9u4SVmGnMge89FoGhpRknD0l+ZyzNrmsX5PK50of+rv/ddc45vM97s28iEuNi+JmHDSHPRziqUak2dtCcFWKcn/NFtrdw2NomLsuFHPVhziw98DWXDazYXWE9O7Ik4TehawbiKTlB6Cg4Wbw5BNQYlJ1tR6wP5aCeaV7H4NZbr8Y+/ezEJ
```
Comment (eolweus) [inline src/demo/example-data/new-layered-map.json:63]: resolve this
```suggestion
                        "type": null,
```

Comment (eolweus) [inline src/demo/example-data/new-layered-map.json:63]: Should be resolved

File: examples/example_new_layered_map_v3.py [hunk 2]
```diff
@@ -513,6 +513,7 @@ def update_colorcsale(colorScale, layer_id):
             global layers
 
             layer_type = get_layer_type(layer_id, layers)
+            print("updating layer with id: ", layer_id)
```
Comment (eolweus) [inline examples/example_new_layered_map_v3.py:516]: Remove prints 🖨

File: src/lib/components/NewLayeredMap/layers/L.imageWebGLOverlay.js [hunk 3]
```diff
@@ -112,8 +112,9 @@ L.ImageWebGLOverlay = L.Layer.extend({
 	    
     	if(options.colorScale && options.colorScale instanceof Object && this.options.colorScale instanceof Object) {
     		options.colorScale = Object.assign(this.options.colorScale, options.colorScale)
-   	}       	   
+   	    }
```
Comment (eolweus) [inline src/lib/components/NewLayeredMap/layers/L.imageWebGLOverlay.js:115]: Not exactly an error, but maybe line up the  } when checking other stuff?

Comment (Andorr) [reply src/lib/components/NewLayeredMap/layers/L.imageWebGLOverlay.js:115]: I will fix these in interns-dev

File: examples/example_new_layered_map_v3.py (hunk 2 above)
Comment (Andorr) [reply examples/example_new_layered_map_v3.py:516]: I will fix these in interns-dev

**Pull request merged**